plugin.video.imdb.trailers/changelog.txt

```diff
@@ -1,5 +1,9 @@
 plugin.video.imdb.trailers:
```

Comment:
no need to submit changelog anymore

Reply:
Thanks, will remove it from next release